extension.js

```diff
@@ -152,15 +152,29 @@ function onHotkeyPressed() {
 }
 
 function enable() {
-  sig_keymap = Keymap.connect('state_changed', onHotkeyPressed);
+  GLib.timeout_add(GLib.PRIORITY_DEFAULT, 1000, () => {
+    const display = Gdk.Display.get_default();
+
+    if (display !== null) {
```

Comment:
Will this part of code loop forever if Gdk.Display.get_default() always return null?

Reply:
Probably yes, the display could be null in something like a headless server.